src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/PartialSuccessTransmissionPolicy.cs

```diff
@@ -14,9 +14,9 @@
 
     internal class PartialSuccessTransmissionPolicy : TransmissionPolicy, IDisposable
     {
-        private readonly JavaScriptSerializer serializer = new JavaScriptSerializer();
+        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(TransmissionPolicyHelpers.SlotDelayInSeconds) };
 
-        private TaskTimer pauseTimer = new TaskTimer { Delay = TimeSpan.FromSeconds(SlotDelayInSeconds) };
+        public int ConsecutiveErrors { get; set; }
```

Comment:
What happens when several unrelated retries occur over time, does it keep getting longer of a delay even if they are unrelated and occur far apart?